keps/sig-node/4381-dra-structured-parameters/kep.yaml

```diff
@@ -44,4 +45,6 @@ disable-supported: true
 
 # The following PRR answers are required at beta release
 metrics:
-  # - my_feature_metric
+  - resource_controller_create_total
+  - resource_controller_create_failures_total
+  - resource controller workqueue with name="resource_claim"
```

Comment:
I tried to search for these metrics but I did not find them in the code base.

Maybe `resourceclaim_controller_create_attempts_total`
and `resourceclaim_controller_create_failures_total`.

I don't know what the last metric corresponds too.

Reply:
Good catch, will update the KEP.

Reply:
About "resource controller workqueue": Those are the `workqueue_*` metrics from https://kubernetes.io/docs/reference/instrumentation/metrics inside the kube-controller-manager.